### DISCUSSION TITLE
Release date for Enhanced Table Kibana Plugin Version compatible with Kibana 7.11.1

### DISCUSSION THREAD

**Discussion by dharmaru** [fbaligand/kibana-enhanced-table#187] [Q&A] [answered]
Hello Fabien,

Regarding your Enhanced Table kibana plugin, may I know when it's version compatible to kibana version 7.11.1 will be released? 

Thanks,
dharm@ru

**Comment by fbaligand** [comment, marked as answer]
Sorry for my late answer. 
Enhanced table is already available for kibana 7.11. 

you can download it here:
https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.11.2
